.github/workflows/ci.yml

```diff
@@ -56,6 +56,19 @@ jobs:
                 pip install --no-build-isolation --no-deps -e .
                 cd ..
 
+            - name: Install imSim
+              run:
+                pip install .
+
+            - name: Install test deps
+              run: |
+                conda install -y pytest pytest-xdist
+                pip install pytest-durations
```

Comment:
The above two are repeated below.  I think you want them there, not here.  But at least don't install them twice.